### DISCUSSION TITLE
Transform MVP Content & Concepts

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#366] [General]
We want to use our mapping (in development) to transform MARC21 (as MARC XML) to LRM/RDA/RDF (RDF/XML) using XSLT. <br>

This discussion is here as a jumping-off point, where we can collectively plan and gather our thoughts. <br>

We need to:

- [ ] Set goals/requirements
- [ ] Assemble a team & assign them to an issue
- [ ] Get started

Theo will lead transformation creation as resident XSLT expert, with support from Crystal, Laura, and Zhuo (and anyone else who is willing to help!) <br>

**Comment by CECSpecialistI** [comment]
@gerontakos @lake44me @pan-zhuo

**Reply by lake44me** [reply]
Quick notes from today's discussion:  I suggested must haves:

1xx
245
260
300

Later thought:  what are essential fixed field values?  Type... etc.

My thought was, if you had a discovery layer based on this - what would be enough info for a user to identify what the data's about.

Then icing would be, 1 or 2 or 3 examples of tricky mapping transformations if we want to show and tell about them.

**Reply by lake44me** [reply]
So, outcome of this redefined discussion should be? 

*  a list of tags/data elements for demonstration purposes as a "proof of concept"?   and perhaps to illustrate some of the preliminary decisions we've made? 
*  perhaps target example records with those tags and perhaps specific conditions? to transform,  
*  a somewhat clear boundary line between "MVP" scope and stuff outside of it, at least for our November target.
*  e.g., if we decide serials are inside the boundary, we'd need 264, but if we leave them out, perhaps not.

**Comment by CECSpecialistI** [comment]
I just identified 80 fields as priorities for the MVP, thinking of print and electronic books, acceptably-detailed, BSR-level cataloging. They're tagged as part of the [MVP milestone](https://github.com/uwlib-cams/MARC2RDA/issues?q=is%3Aissue+is%3Aopen+milestone%3A%22MVP+for+Transformation%22). The milestone tags can and should be changed by anyone who disagrees with the choices I made, which were subjective. If you end up removing one of the ones I identified, please replace the MVP milestone with the PCC RDA BSR milestone rather than just leaving the issue untethered to any particular milestone :)

**Reply by CECSpecialistI** [reply]
@gerontakos @pan-zhuo
